apiserver/facades/client/client/client.go

```diff
@@ -526,16 +526,19 @@ func (c *Client) ModelInfo() (params.ModelInfo, error) {
 	if err != nil {
 		return params.ModelInfo{}, err
 	}
+
 	info := params.ModelInfo{
-		DefaultSeries: config.PreferredSeries(conf),
-		CloudTag:      names.NewCloudTag(model.Cloud()).String(),
-		CloudRegion:   model.CloudRegion(),
-		ProviderType:  conf.Type(),
-		Name:          conf.Name(),
-		Type:          string(model.Type()),
-		UUID:          model.UUID(),
-		OwnerTag:      model.Owner().String(),
-		Life:          params.Life(model.Life().String()),
+		DefaultSeries:  config.PreferredSeries(conf),
+		CloudTag:       names.NewCloudTag(model.Cloud()).String(),
+		CloudRegion:    model.CloudRegion(),
+		ProviderType:   conf.Type(),
+		Name:           conf.Name(),
+		Type:           string(model.Type()),
+		UUID:           model.UUID(),
+		OwnerTag:       model.Owner().String(),
+		Life:           params.Life(model.Life().String()),
+		ControllerUUID: state.ControllerTag().String(),
```

Comment:
I do question whether or not the ControllerUUID is something that the user should really be caring about, but it probably can't hurt that much.

Reply:
That's a fair point, but one that I didn't really question as it was already part of the struct. Happy to pull it out - but perhaps we should file another ticket and see if there is anyone relying on it?